ot/da.py

```diff
@@ -845,7 +825,7 @@ def fit(self, xs, ys, xt, reg=1, eta=1, ws=None, wt=None, norm=None,
         self.wt = wt
 
         self.M = dist(xs, xt, metric=self.metric)
-        self.normalizeM(norm)
+        self.M = cost_normalization(self.M, norm)
```

Comment:
- @aje : can you handle the case where one does not want to normalise the cost matrix ?
- should we do that with a `if` in the `.fit` or in the `cost_normalization` function ?

Reply:
If you give it None is is by default the function will already return C. 

Seems OK to me.

Reply:
I could add the check
```
if self.norm is not None:
```
but this also seems ok to me